modules/microshift-install-bootc-publish-image.adoc

```diff
@@ -28,15 +28,15 @@ $ sudo podman login "${REGISTRY_URL}" # <1>
 
 . Publish the image by running the following command:
 +
-[source,terminal]
+[source,terminal,subs="attributes+",subs="quotes+"]
 ----
-REGISTRY_IMG=<myorg/mypath>/"${IMAGE_NAME}" # <1> <2>
-IMAGE_NAME=<microshift-4.17-bootc> # <3>
+REGISTRY_IMG=_<myorg/mypath>_/"${IMAGE_NAME}" # <1>
```

Comment:
This is not rendered correctly. Has `<em>` extra tags.

Reply:
Yes, the asciidoc syntax isn't working as expected there. Trying something else.